### PR TITLE
[RFR] Analyze Coolstore app off Quarkus branch - Source and dependencies

### PR DESCRIPTION
**Pull request by nachandr**

### CODE REVIEW

**Review by aufi — APPROVED** [review]
I cannot validate if issues analysis results are fully correct, but from quick look, looks OK.

Better ask someone contributing to the application repo itself: https://github.com/konveyor-ecosystem/coolstore

**Comment by nachandr** [comment]
Hi @jmle , @jwmatthews , Could you let us know if the analysis results look good ?

**Comment by nachandr** [comment]
Note : The test for the analysis of the Coolstore  app is a tier 0 test and the test-tier0 check failed because Tackle couldn't be installed.

File: analysis/tc_coolstore_deps_quarkus.go [hunk 1]
```diff
@@ -0,0 +1,1082 @@
...
+				Effort: 7,
+				RuleSet: "cloud-readiness",
+				Rule: "localhost-http-00001",
+				Incidents: []api.Incident{
+					{
+						File: "/shared/source/coolstore/src/main/resources/application.properties",
+						Line: 10,
+						Message: "The app is trying to access local resource by HTTP, please try to migrate the resource to cloud",
+					},
+				},
```
Comment (jmle) [inline analysis/tc_coolstore_deps_quarkus.go:39]: With `cloud-readiness` and `jakarta-ee` we should see more results than this. There is something going on, but not sure if it's the test itself or something else. It looks as if the java provider wasn't starting up for some reason. This particular issue belongs to a rule that uses the built-in provider, not the java provider.

**Review by jmle — CHANGES_REQUESTED** [review]
We need to find out if the lack of issues is due to a bad execution of the tests, or a bug in the system.

**Review by jmle — APPROVED** [review]
Looks good @nachandr , it makes sense that we're only getting one issue, since the `quarkus` branch has already been migrated.

**Pull request merged**